gpt4all-chat/qml/ChatView.qml

```diff
@@ -2188,7 +2131,7 @@ Rectangle {
 
             MyFileDialog {
                 id: fileDialog
-                nameFilters: ["Excel files (*.xlsx)"]
+                nameFilters: ["Excel files (*.xlsx)", "Text files (*.txt)", "Markdown files (*.md)"]
```

Comment:
The default filter doesn't show text files, which hurts discoverability of this feature. I suggest:
```suggestion
                nameFilters: ["All Supported Files (*.txt *.md *.xlsx)", "Text Files (*.txt *.md)", "Excel Worksheets (*.xlsx)"]
```